sdk/servicebus/service-bus/test/samples.spec.ts

```diff
@@ -20,13 +30,15 @@ describe("Ensure typescript samples use published package", function(): void {
     }
   }
 
-  it("Ensure getStarted samples use published package", () => {
-    const folder = path.join(__dirname, "../../samples/typescript/gettingStarted");
-    testSamples(folder);
+  it("Ensure TypeScript samples use published package", async () => {
+    const pattern = path.join(__dirname, "../../samples/typescript/src") + path.sep + "**/*.ts";
```

Comment:
Just curious, is `path.sep` important here? If so, should it be used in the tail-end of the glob pattern as well?
```typescript
`**${path.sep}*.ts`
```

Reply:
actually now that you mention it I think glob requires forward slashes irrespective of the platform

looks like glob is causing karma tests to fail anyway... after going through this I've realized I don't understand how this test _ever_ ran successfully in karma...